assume_unchanged.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
```

Comment:
This file would be better off in a dedicated scripts directory so it doesn't clutter the top level